translatable/tests/unitary/translation_collection.rs

```diff
@@ -1,6 +1,6 @@
 use std::collections::HashMap;
 
-use toml::Table;
+use toml_edit::DocumentMut;
```

Comment:
For all the `DocumentMut` references too there is [`ImDocument`](https://docs.rs/toml_edit/latest/toml_edit/struct.ImDocument.html) which is a non mutable version of `DocumentMut`.